### PR TITLE
Adding dask-geopandas

### PR DESCRIPTION
**Pull request by martinfleis**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by martinfleis** [comment]
@jorisvandenbossche @jsignell I have listed you as maintainers of this recipe, can you confirm you're fine with that?

**Comment by jorisvandenbossche** [comment]
I am OK to be listed as maintainer!

**Comment by jorisvandenbossche** [comment]
Regarding the LICENSE file, it seems to be correctly included in the sdist, so then it doesn't need to be added here as physical file?

**Comment by martinfleis** [comment]
True, will remove it.

**Comment by martinfleis** [comment]
CI actually failed without an attached license.

**Comment by jorisvandenbossche** [comment]
That might be because it's named `LICENSE` and not `LICENSE.txt` ?

File: recipes/dask-geopandas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - distributed >=2.2.0
+
+test:
+  imports:
+    - dask_geopandas
+
+about:
+  home: https://github.com/geopandas/dask-geopandas
+  license: BSD-3-Clause
+  license_file: LICENSE.txt
```
Comment (jorisvandenbossche) [inline recipes/dask-geopandas/meta.yaml:33]:
```suggestion
  license_file: LICENSE
```

**Comment by jsignell** [comment]
Yeah I am happy to be listed.

**Comment by martinfleis** [comment]
ping @conda-forge/help-python, this PR is ready

**Review by jakirkham — APPROVED** [review]

File: recipes/dask-geopandas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.0a2" %}
```
Comment (jakirkham) [inline recipes/dask-geopandas/meta.yaml:1]: This says it is an alpha release. Do we have a final release?

File: recipes/dask-geopandas/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    - geopandas >=0.9
+    - dask >=2.2.0
+    - distributed >=2.2.0
+
+test:
+  imports:
+    - dask_geopandas
+
+about:
+  home: https://github.com/geopandas/dask-geopandas
```
Comment (jakirkham) [inline recipes/dask-geopandas/meta.yaml:31]: Normally this refers to the `url` in `setup.py`, but interestingly that page doesn't seem to exist atm

https://github.com/geopandas/dask-geopandas/issues/41

File: recipes/dask-geopandas/meta.yaml (hunk 2 above)
Comment (jorisvandenbossche) [reply recipes/dask-geopandas/meta.yaml:1]: It's a first ever release, and I think the idea for now was to keep doing alpha releases for a while (but already make it easier for people to install it with conda-forge). 
But if conda-forge prefers non-pre-release numbers, we can consider changing the version number, doesn't matter too much.

**Comment by jakirkham** [comment]
Sorry missed a couple things when looking initially. Asked about them above

**Comment by jorisvandenbossche** [comment]
Thanks @jakirkham for taking a look!

File: recipes/dask-geopandas/meta.yaml (hunk 2 above)
Comment (jakirkham) [reply recipes/dask-geopandas/meta.yaml:1]: That should be fine. We have [this wording in CFEP 5]( https://github.com/conda-forge/cfep/blob/master/cfep-05.md#exceptions ), which should allow for this

Comment (jorisvandenbossche) [reply recipes/dask-geopandas/meta.yaml:1]: Indeed, there has not yet been a non-alpha release before, so we should fall under this exception rule

File: recipes/dask-geopandas/meta.yaml (hunk 3 above)
Comment (jorisvandenbossche) [reply recipes/dask-geopandas/meta.yaml:31]: I opened a PR on the dask-geopandas side to fix this

Comment (jakirkham) [reply recipes/dask-geopandas/meta.yaml:31]: Thanks Joris! I don't think we need to wait for that. Let's resolve that issue asynchronously

**Comment by jakirkham** [comment]
Thanks all! 😄

For the remaining issue (`home` url), we can handle this asynchronously

**Pull request merged**